eth-contracts/contracts/ClaimsManager.sol

```diff
@@ -1,11 +1,12 @@
 pragma solidity ^0.5.0;
+
 import "./Staking.sol";
 import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC20/ERC20Mintable.sol";
 import "./ServiceProviderFactory.sol";
-/// @notice ERC20 imported via Staking.sol
-/// @notice SafeERC20 imported via Staking.sol
-/// @notice Governance imported via Staking.sol
-/// @notice SafeMath imported via ServiceProviderFactory.sol
+import "./InitializableV2.sol";
+import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC20/SafeERC20.sol";
+import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC20/ERC20.sol";
+import "./Governance.sol";
```

Comment:
for some reason old way with indirect imports stopped compiling. i'll try to fix this

Reply:
fixed